india_stocks_api/internal/fivepaisa/api/funds.py

```
@@ -0,0 +1,90 @@
+            - m2mrealized: Total booked P&L
+            - utiliseddebits: Utilized margin
+    """
+    # Get credentials
+    try:
+                
+        creds=get_credentials("fivepaisa")
+        api_key = creds["api_key"]
+        client_id = creds["clientcode"]
+
+    except ValueError:
+        raise ValueError("Failed to get credentials. Please check your configuration.")
```

Ruff (0.14.14)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)